react-youtube-adaptive-loading/src/App.js

```diff
@@ -41,7 +41,9 @@ const App = ({ youtubeLibraryLoaded, location }) => {
   } = useLiteModeDebugging();
 
   useEffect(() => {
-    loadYoutubeApi();
+    if (!YOUTUBE_API_DEV_MODE) {
```

Comment:
For later: due to YouTube API restrictions we have landed on a model where we can optionally use static data for testing or live data via an API key if we prefer.